scripts/release.sh

```diff
@@ -53,13 +53,24 @@ fi
 
 # Build Docker server image
 if [[ -z "$SKIP_DOCKER" ]]; then
-  docker build -t $docker_name:$version .
-  docker build -t $docker_name:$version-standalone --build-arg STANDALONE=1 .
+  docker build -t $docker_name:$version -f amd64.Dockerfile .
+  docker build -t $docker_name:$version-standalone-amd64 -f amd64.Dockerfile --build-arg STANDALONE=1 .
+  docker build -t $docker_name:$version-standalone-arm32v7 -f arm32v7.Dockerfile .
+
+  # We need to create the multi arch image for -standalone
+  # Make sure experimental docker cli feature is on: echo "{ \"experimental\": \"enabled\" }" >> $HOME/.docker/config.json
+  docker manifest create --amend $docker_name:$version-standalone $docker_name:$version-standalone-amd64 $docker_name:$version-standalone-arm32v7
+  docker manifest annotate $docker_name:$version-standalone $docker_name:$version-standalone-amd64 --os linux --arch amd64
+  docker manifest annotate $docker_name:$version-standalone $docker_name:$version-standalone-arm32v7 --os linux --arch arm --variant v7
+
   docker tag $docker_name:$version $docker_name:latest
   docker tag $docker_name:$version-standalone $docker_name:standalone
   # we shouldn't push docker this early in the script, but the docker image hash is not available until we do
   # and is needed for the SHA256SUMS file. https://groups.google.com/forum/#!topic/docker-user/PvAcxDrvP30
   # https://github.com/moby/moby/issues/16482 https://github.com/docker/distribution/issues/1662
+  docker push $docker_name:$version-standalone-arm32v7
+  docker push $docker_name:$version-standalone-amd64
+  docker push $docker_name:$version-standalone
```

Comment:
`docker push <name>` without a version pushes all of them

Reply:
I was wondering if it works, as you want to publish the manifest AFTER the arch specific images.
But should be OK.